source/Playnite.DesktopApp/ViewModels/GameEditViewModel.cs

```diff
@@ -921,6 +925,15 @@ List<Guid> consolidateIds(SelectableDbItemList selectionList, List<Guid> origina
                     game.OverrideInstallState = EditingGame.OverrideInstallState;
                 }
 
+                if (appSettings.GameSortingNameAutofill && string.IsNullOrEmpty(game.SortingName))
+                {
+                    var sortingName = sortableNameConverter.Convert(game.Name);
+                    if (sortingName != game.Name)

```

Comment:
Can SortableNameConverter change casing? I didn't check the code, but if so, then I would do case insensitive comparison here instead.